22-053/sections/01-introduction.adoc

```diff
@@ -1,14 +1,14 @@
 [[ug_introduction_section]]
 == Introduction
 
-An increasing number of cities and companies are building virtual 3D city models for different application areas like urban planning, mobile telecommunication, disaster management, 3D cadastre, tourism, vehicle and pedestrian navigation, facility management and environmental simulations. Furthermore, in the implementation of the European Environmental Noise Directive (END, 2002/49/EC) 3D geoinformation and 3D city models play an important role.
+There are many systems and applications that need to have information about locations in the world. For example, such "point of interest" (POI) data is needed in navigation systems, mapping, geocaching, location-based social networking games, and augmented reality browsers, among many others.
```

Comment:
Should there be a pointer to use cases we have examined?

Reply:
Yes, this section needs more work and mentioning some of the use cases we've examined would be good. Not changing this right now, however.